lk2nd/device/dts/msm8953/sdm632-fairphone-fp3.dts

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+#include <skeleton64.dtsi>
+#include <lk2nd.dtsi>
+
+/ {
+	model = "Fairphone 3";
+	qcom,msm-id = <QCOM_ID_SDM632 0>;
+	qcom,pmic-id = <0x10016 0x25 0x00 0x00>;
+	qcom,board-id = <8 3>;
+	__symbols__ {};
+	chosen {};
+	memory {};
```

Comment:
```suggestion
```

they are included in skeleton64.dtsi:
https://github.com/msm8916-mainline/lk2nd/blob/6461706bb92978923474ac670ef38249dbba4e95/lk2nd/device/dts/skeleton64.dtsi#L9-L14